### PR TITLE
PUD-1458 - set recall type / new feature for standard recalls

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: cypress/integration/inCustodyFixed.feature [hunk 1]
```diff
@@ -11,6 +11,7 @@ Feature: In custody
     And Maria enters the licence name
     And Maria enters the pre-convictions name
     And Maria confirms the person is in custody
+    And Maria confirms the recall type as Fixed term
     And Maria submits the date and email of the recall request received from probation service
     And Maria submits the sentence, offence and release details
     And Maria submits the police contact details
```
Comment (philsegal) [inline cypress/integration/inCustodyFixed.feature:17]: should we be confirming the recall type in CYA ?

Comment (jonnywyatt) [reply cypress/integration/inCustodyFixed.feature:17]: it just means 'sets', same as in the line above it. I've changed to 'sets'

**Review by philsegal — CHANGES_REQUESTED** [review]
Check recall type on CYA for Fixed term

File: cypress/integration/inCustodyFixed.feature (hunk 1 above)
Comment (philsegal) [reply cypress/integration/inCustodyFixed.feature:17]: Yep, I meant, we should add a line somewhere, similar to what we have in Standard? e.g. verify that the value is shown on CYA page? Or do we already have this?

Comment (jonnywyatt) [reply cypress/integration/inCustodyFixed.feature:17]: I've verified on CYA now

**Pull request merged**